### ISSUE TITLE
[Effect Suggestion] SA Sightseeing

### ISSUE THREAD

**Issue by aimlessB** [gta-chaos-mod/ChaosModV#1495]
**Name of effect**
SA Sightseeing, Sightseeing

**Duration if timed effect (long or short timed?)**
Short

**Effect Description**
For 20 Seconds, teleport the player to a completely random location on the map every 5 Seconds, after the 20 Seconds, the player stays in the last location.

**Comment by Last0xygen** [comment]
Duplicate https://github.com/gta-chaos-mod/ChaosModV/issues/1019
… did you really open it in a new request just to push it?

**Comment by aimlessB** [comment]
No, because I didn't know how to edit on the phone and I really liked the idea so I wanted to specify what the effect should do.

**Comment by aimlessB** [comment]
I thought it would be alright if I closed the former version.
